base_conditional_image/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
This file should be generated automatically.
You have to create a readme directory with every sections of the `README.rst`.

Reply:
It's a V9 branch, no module are using this pattern. I'll switch to it when I migrate the module to V10 and later

Reply:
I think it's not related to the version.
No module in v9.0 using this pattern because this pattern is quite recent

Reply:
AFAIK it's a recommendation, and it's not mandatory. And the readme is really explicit.
@acsonefho Do you think your comment on this point can be dismissed?

Reply:
@leemannd Some say that's required and others just a recommendation.
On the documentation (on maintainer-tools repo), the readme file is mandatory and the way to generate it is with the `readme` directory and some files.
I think that someone should take a final decision about that.
I'll not "block" the MR for this of course!